yt/utilities/linear_interpolators.py

```diff
@@ -21,53 +92,52 @@ def __init__(self, table, boundaries, field_names, truncate=False):
             If False, an exception is raised if the input values are
             outside the bounds of the table.  If True, extrapolation is
             performed.
+        store_table: bool
+            If False, only the shape of the input table is stored and
+            a full table must be provided when calling the interpolator.
```

Comment:
The default value should be explicit. (This comment applies, of course, to everywhere this is copied)

Reply:
re-wrote this to note the default